src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/commands.py

```diff
@@ -33,3 +34,9 @@ def transform_account_list(result):
 cli_command(__name__, 'account clear', _custom_module + 'account_clear')
 cli_command(__name__, 'account list-locations', _custom_module + 'list_locations')
 cli_command(__name__, 'account get-access-token', _custom_module + 'get_access_token')
+cli_command(__name__, 'account lock create', 'azure.cli.command_modules.resource.custom#create_lock')
+cli_command(__name__, 'account lock delete', 'azure.cli.command_modules.resource.custom#delete_lock')
```

Comment:
We should think twice here for the cross command module reference which we should alway avoid.
Also, it is questionable to expose a general resource lock command under `az account lock`. The goal here is for simplified user experience, so we should have a custom method to only let you lock at subscription level